cpp_exercises/day3/phoneBook/tests/tst_phonebook.cpp

```diff
@@ -67,6 +73,21 @@ void testPhoneBook::findTest()
     QCOMPARE(contact, nullptr);
 }
 
+void testPhoneBook::insertionOperatorTest() {
+    PhoneBook phoneBook;
+    fill(phoneBook);
+
+    std::stringstream out;
+    out << phoneBook;
+    const char* resultExpected = "1) Calà Gerry\n"
```

Comment:
In C++11 puoi anche definire delle raw string literals: https://en.cppreference.com/w/cpp/language/string_literal
Così ti risparmi i `\n` in giro